### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by v3ss0n** [pyinfra-dev/pyinfra#951] [Q&A] [answered]
Just  trying and then the project  seems  inactive. Is this project dead?

**Comment by Fizzadar** [comment, marked as answer]
Definitely not dead! Two weeks since last commit isn't exactly long!

**Comment by v3ss0n** [comment]
Thanks , we are using it and enjoying. found a few bugs in examples gonna push more fixes.
Also I can help with LXD integration, since current lxd integration is just initial. since we had used LXD API extensively in some project.
